docs/_articles/en/soql/soql-builder.md

```diff
@@ -60,7 +67,27 @@ As you build your query, watch SOQL Builder display the query syntax while it si
 
 You can select objects and fields from the drop-down list, or type to narrow the list results. You can select an object or a field only once. If a value is already selected, it doesn't appear in the drop-down or search results.
 
-![Build your query](./images/soql-builder.gif)
+![Build your query](./images/soql-builder-build-a-query.gif)
+
+<!-- ekapner, 2/18/2021 - new gif with new name -->
+
+### Filter with the LIKE Operator <!-- ekapner, copy to JA version -->
+
+When filtering your results, you can narrow and target those results even further by using the LIKE operator to match partial text strings using wildcards. This query returns only last names that start with with `appl`.
+
+```
+SELECT AccountId, FirstName, lastname
+FROM Contact
+WHERE lastname LIKE 'appl%'
+```
+
+You can build your own filter using LIKE, or you can select one of these pre-built options.
+
+- starts with
+- ends with
+- contains
+
+### View COUNT Results <!-- ekapner, 2/19, new title here -->
```

Comment:
Nice balance here between linking the LIKE variations and too much explanation.

Reply:
+1